WORKSPACE

```diff
@@ -0,0 +1,40 @@
+# To update TensorFlow to a new revision.
+# 1. Update the 'git_commit' args below to include the new git hash.
+# 2. Get the sha256 hash of the archive with a command such as...
+#    curl -L https://github.com/tensorflow/tensorflow/archive/<git hash>.tar.gz | sha256sum
+#    and update the 'sha256' arg with the result.
+# 3. Request the new archive to be mirrored on mirror.bazel.build for more
+#    reliable downloads.
+load("//build_rules:repo.bzl", "tensorflow_http_archive")
+
+tensorflow_http_archive(
```

Comment:
Please add a comment that the tensorflow version here must match the swift repo.